timeboost-types/src/bundle.rs

```diff
@@ -0,0 +1,463 @@
+use alloy::consensus::TxEnvelope;
+
+use alloy_rlp::Decodable;
+use alloy_signer::{Error, SignerSync, k256::ecdsa::SigningKey};
+use alloy_signer_local::PrivateKeySigner;
+use committable::{Commitment, Committable, RawCommitmentBuilder};
+use serde::{Deserialize, Serialize};
+use timeboost_crypto::KeysetId;
+
+use crate::{Bytes, Epoch, SeqNo};
+
+const DOMAIN: &str = "TIMEBOOST_BID";
+
+#[derive(Debug, Clone)]
+pub enum BundleVariant {
+    Regular(Bundle),
+    Priority(PriorityBundle<Signed>),
+}
+
+#[derive(Debug, Clone, PartialEq, Eq, Hash, PartialOrd, Ord, Serialize, Deserialize)]
+pub struct Bundle {
```

Comment:
Perhaps this should be named `RegularBundle` and `BundleVariant` can become `Bundle`?